tags_push/download.py

```diff
@@ -1,21 +1,12 @@
 from exercise_utils.cli import run_command
-from exercise_utils.git import tag, push
+from exercise_utils.git import tag
 from exercise_utils.gitmastery import create_start_tag
 
-__resources__ = {}
-
-REMOTE_NAME = "production"
-TAG_1_NAME = "v1.0"
-TAG_2_NAME = "v2.0"
-TAG_DELETE_NAME = "beta"
-TAG_2_MESSAGE = "First stable roster"
-
 def setup(verbose: bool = False):
-    create_start_tag(verbose)
-    run_command(["git", "remote", "rename", "origin", REMOTE_NAME], verbose)
-    tag(TAG_DELETE_NAME, verbose)
-    push(REMOTE_NAME, "--tags", verbose) # somewhat hacky, maybe use run_command instead
-    run_command(["git", "tag", "-d", TAG_DELETE_NAME], verbose)
+    run_command(["git", "remote", "rename", "origin", "production"], verbose)
+    tag("beta", verbose)
+    run_command(["git", "push", "production", "--tags"], verbose)
+    run_command(["git", "tag", "-d", "beta"], verbose)
 
-    run_command(["git", "tag", TAG_1_NAME, "HEAD~4"], verbose)
-    run_command(["git", "tag", "-a", TAG_2_NAME, "HEAD~1", "-m", f"\"{TAG_2_MESSAGE}\""], verbose)
+    run_command(["git", "tag", "v1.0", "HEAD~4"], verbose)
+    run_command(["git", "tag", "-a", "v2.0", "HEAD~1", "-m", f"\"{"First stable roster"}\""], verbose)
```

Comment:
```suggestion
    run_command(["git", "tag", "-a", "v2.0", "HEAD~1", "-m", "First stable roster"], verbose)
```

The quotes in the code from the exercise review were for the `-m` parameter, it's not necessary when passed this way!